source/reference/operator/aggregation/geoNear.txt

```diff
@@ -204,8 +204,147 @@ When using :pipeline:`$geoNear`, consider that:
 - Starting in version 4.2, :pipeline:`$geoNear` no longer has a default
   limit of 100 documents.
 
-Example
--------
+- Starting in MongoDB 5.1, the ``near`` parameter supports the 
+  :ref:`let option <geoNear_let_example>` and
+  :ref:`bound let option <geoNear_bounded_let_example>`.
+
+Examples
+--------
+
+.. _geoNear_let_example:
+
+$geoNear with the ``let`` option
```

Comment:
I think the maximum and minimum distance examples should still be the first examples in this section. I'm worried that if we put the `let` option first, we're introducing a fairly abstract / complex example as the first one on the page. Let's make sure that the simplest, most straight-forward examples appear first when possible.

Reply:
Updated.